app/components/certificate/loading.tsx

```diff
@@ -0,0 +1,18 @@
+import { Center, Flex, Box, Image, Heading } from '@chakra-ui/react';
+
+import undrawSvg from 'img/undraw_processing_re_tbdu.svg';
```

Comment:
We're still missing the license on this image, it needs to be included in our repo somehow (maybe in `img/*`)